config/crd/kustomization.yaml

```diff
@@ -9,7 +9,7 @@ resources:
   - bases/metallb.io_l2advertisements.yaml
   - bases/metallb.io_bgpadvertisements.yaml
   - bases/metallb.io_communities.yaml
-  - bases/metallb.io_servicel2statuses.yaml
+# - bases/metallb.io_servicel2statuses.yaml TODO uncomment when metallb support is completed
```

Comment:
Is there a plan/issue for that?

Reply:
yep, that's being worked out in https://github.com/metallb/metallb/pull/2351

Reply:
I would personnally add the issue in the comment or remove all the way (but no strong opinion)

Reply:
I thought about that, but we'll remember as when we'll bump metallb we'll bump the corresponding tests which will fail because the crd is not there.